src/main/java/tp/Ward.java

```diff
@@ -8,9 +8,9 @@
     protected Doctor doctor;
     protected Patient patient;
     protected Nurse nurse;
-    protected String wardNumber;
-    
-    public Ward(Doctor doctor, Patient patient, Nurse nurse, String wardNumber) {
+    protected int wardNumber;
+
+    public Ward(Doctor doctor, Patient patient, Nurse nurse, int wardNumber) {
```

Comment:
It's good that you changed wardNumber to integer instead of string! It will help to prevent the user from entering meaningless input.